### PR TITLE
pin latest state comment

### LINKED ISSUE #1177: Pin latest state comment
- duplicate if comments more than 1

### PR DESCRIPTION
**Pull request by DenisVorop**
## PR includes

- [x] Feature

## Related issues

Resolve #1177 

## QA Instructions, Screenshots, Recordings
  
Снимок экрана 2023-07-13 в 09 48 27

### CODE REVIEW

File: src/components/CommentFixed.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+            <StyledCommentCard>
+                <CardInfo onClick={onChangeTypeDate}>
+                    <Link inline>{author?.name}</Link> —{' '}
+                    <Link inline href={`#comment-${id}`}>
+                        <RelativeTime isRelativeTime={isRelativeTime} date={createdAt} hover />
+                    </Link>
+                </CardInfo>
+
+                <CardComment>
```
Comment (9teen90nine) [inline src/components/CommentFixed.tsx:92]: this looks a lot like a copy-paste of a comment component with some logic stripped of
you need either combine them into a single one or somehow reference one from another

**Review by 9teen90nine — CHANGES_REQUESTED** [review]
We don't make copies!

File: src/components/CommentView/CommentView.tsx [hunk 2]
```diff
@@ -104,25 +107,48 @@ const StyledCommentCard = styled(Card)<{ isNew?: boolean }>`
...
+        color={item.color}
+        focused={cursor === index}
+        icon={item.icon}
+        onClick={item.onClick}
+    >
+        {item.label}
+    </MenuItem>
+);
+
+const iconRenderCondition = (isPinned: boolean, image?: User['image'], email?: User['email']) =>
```
Comment (9teen90nine) [inline src/components/CommentView/CommentView.tsx:143]: this too

**Review by 9teen90nine — CHANGES_REQUESTED** [review]
you should create new files for support components and make them components, not render functions

**Review by DenisVorop — COMMENTED** [review]
nothing

**Pull request closed without merging**